lib/models/application.coffee

```diff
@@ -694,6 +694,128 @@ getApplicationModel = (deps, opts) ->
 			throw err
 		.asCallback(callback)
 
+	###*
+	# @summary Check if rolling release is enabled for specific application
+	# @name isRollingReleaseEnabled
+	# @public
+	# @function
+	# @memberof resin.models.application
+	#
+	# @param {String|Number} nameOrId - application name (string) or id (number)
+	# @returns {Promise}
+	#
+	# @example
+	# resin.models.application.isRollingReleaseEnabled('MyApp').then(function(isEnabled) {
+	# 	console.log(isEnabled);
+	# });
+	###
+
+	exports.isRollingReleaseEnabled = (nameOrId, callback) ->
+		exports.get(nameOrId, $select: 'should_track_latest_release')
+		.get('should_track_latest_release').asCallback(callback)
+
+	###*
+	# @summary Set application to particular commit
+	# @name setApplicationToRelease
+	# @public
+	# @function
+	# @memberof resin.models.application
+	#
+	# @param {String|Number} nameOrId - application name (string) or id (number)
+	#
+	# @description this method requires rolling release to be disabled prior to call
+	#
+	# @returns {Promise}
+	#
+	# @example
+	# resin.models.application.disableRollingRelease('MyApp').then(
+	# 	resin.models.application.isRollingReleaseEnabled('MyApp').then(function(isEnabled) {
+	# 		console.log(isEnabled);
+	# 	})
+	# );
+	###
+
+	exports.setApplicationToRelease = (nameOrId, fullCommitHash, callback) ->
```

Comment:
Can we rename this to `fullReleaseHash`? Just to be clearer that this is the release identifier.

Not a big deal right now, but it'll be useful to make sure we're talking in terms of releases. Later on, we're probably going to want to also allow release ids here too, and in future more and more releases will come from `resin push`, not actual commits.